src/Microsoft.Diagnostics.Monitoring.EventPipe/ProcessInfo/EventProcessInfoPipelineSettings.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Monitoring.EventPipe
+{
+    internal class EventProcessInfoPipelineSettings : EventSourcePipelineSettings
+    {
+        public Func<string, CancellationToken, Task> CommandLineCallback { get; set; }
```

Comment:
To match the pattern of the other pipelines, shouldn't this be a parameter of the EventProcessInfoPipeline ctor?